src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITests_12_AntigenTestProfile.swift

```diff
@@ -0,0 +1,142 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+
+class ENAUITests_12_AntigenTestProfile: XCTestCase {
+	
+	// MARK: - Overrides
+	
+	override func setUp() {
+		super.setUp()
+		continueAfterFailure = false
+		app = XCUIApplication()
+		app.setDefaults()
+		app.launchArguments.append(contentsOf: ["-isOnboarded", "YES"])
+		app.launchArguments.append(contentsOf: ["-setCurrentOnboardingVersion", "YES"])
+		app.launchArguments.append(contentsOf: ["-antigenTestProfileInfoScreenShown", "NO"])
+		app.launchArguments.append(contentsOf: ["-removeAntigenTestProfile", "YES"])
+	}
+	
+	// MARK: - Internal
+
+	var app: XCUIApplication!
+	
+	// MARK: - Tests
+	
+	func test_FIRST_CreateAntigenTestProfile_THEN_DeleteProfile() throws {
+		
+		app.launch()
+		
+		/// Home Screen
+
+		let registerTestButton = try XCTUnwrap(app.buttons[AccessibilityIdentifiers.Home.submitCardButton])
+		registerTestButton.tap()
+		
+		/// Register Test Screen
+
+		let createProfileButton = try XCTUnwrap(app.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Profile.createProfileTile_Description])
+		XCTAssertTrue(createProfileButton.waitForExistence(timeout: .short))
+		createProfileButton.tap()
+		
+		/// Antigen Test Information Screen
+		
+		// header image exists
+		XCTAssertTrue(app.images[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.imageDescription].exists)
+		// title exists
+		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.descriptionTitle].exists)
+		// subtitle exists
+		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.descriptionSubHeadline].exists)
+		// legal text exists
+		XCTAssertTrue(app.cells[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.acknowledgementTitle].exists)
+		
+		let dataPrivacyButton = try XCTUnwrap(app.cells[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.dataPrivacyTitle])
+		dataPrivacyButton.tap()
+		
+		/// Legal Text Screen
+		
+		XCTAssertTrue(app.cells[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.dataPrivacyTitle].exists)
+		
+		let backButton = try XCTUnwrap(app.navigationBars.buttons.element(boundBy: 0))
+		backButton.tap()
```

Comment:
add here before the tap `backButton.waitForExistence()`

Reply:
Why do we need this?